package.json

```diff
@@ -42,31 +42,32 @@
     "rules"
   ],
   "devDependencies": {
```

Comment:
As there is an security update, https://github.com/briza-insurance/illogical/pull/195, we want to release this ... i.e. bump the library version, the same as for wegood, https://github.com/briza-insurance/wegood/pull/165/files